src/components/Review.jsx

```diff
@@ -100,12 +108,29 @@ const Review = () => {
 						<p>{comment.body}</p>
 					</div>
 				))}
-				<form className="review-card-comment" onSubmit={handleSubmit}>
-					<label htmlFor="comment">
-						<h4>Post a comment:</h4>
-					</label>
-					<input name="comment" type="text" />
-				</form>
+				{user.username ? (
+					<form onSubmit={handleSubmit}>
+						<div className="review-card-comment">
+							<label htmlFor="comment">
+								<img src={user.avatar_url} alt="" height="65px;" />
+							</label>
+							<input name="comment" type="text" />
```

Comment:
This input field isn't a controlled component at the moment. The value of this field should be handled by React and updated with an onChange handler